packages/tldraw/src/components/ToolsPanel/HelpPanel.tsx

```diff
@@ -32,9 +32,9 @@ export function HelpPanel() {
 
   return (
     <Popover.Root>
-      <PopoverAnchor dir="ltr" debug={isDebugMode}>
+      <PopoverAnchor dir="ltr" debug={isDebugMode} side={side} bp={breakpoints}>
         <Popover.Trigger dir="ltr" asChild>
-          <HelpButton side={side} debug={isDebugMode} bp={breakpoints}>
+          <HelpButton debug={isDebugMode}>
```

Comment:
Hi @judicaelandria,

You keep debug prop and remove debug variant.
On build this throw typescript error